src/agent/debuglet.ts

```diff
@@ -392,7 +392,7 @@ export class Debuglet extends EventEmitter {
     //       named `response`) is not used.
     metadata.project(
         'project-id',
-        function(err: Error, _: http.ServerResponse, metadataProject: string) {
+        function(err: Error, _res: http.ServerResponse, metadataProject: string) {
```

Comment:
The current linting setup allows leading underscores, but we should determine if we want leading and/or trailing underscores.  In the `StateResolver` class we also use trailing underscores.  We should determine a standard approach, but we can do that in a follow-up PR.